### PR TITLE
Add a useless file.

### PR DESCRIPTION
**Pull request by zenhack**
I'm giving a tutorial on our workflow.

### CODE REVIEW

**Comment by gsilvis** [comment]
This should say 'bar baz' instead

File: bar [hunk 1]
```diff
@@ -0,0 +1 @@
+foo
```
Comment (zenhack) [inline bar:1]: What's going on here? I don't understand what you're trying to do.

**Comment by zenhack** [comment]
Can you go back and make this file less useless?

**Comment by gsilvis** [comment]
[disclaimer: this was an example pull request for a demonstration]

**Pull request closed without merging**